pkg/server/container_create.go

```diff
@@ -222,6 +222,12 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 	// TODO(random-liu): [P1] Set selinux options.
 
 	// TODO(random-liu): [P1] Set user/username.
+	// NOTE(zzh): only set the user id.
```

Comment:
Just pure comment is ok here. No need to add `NOTE`.

You can add TODO for the user name, and remove the previous TODO.